tensorflow/python/training/checkpoint_management.py

```diff
@@ -494,7 +494,7 @@ class CheckpointManager(object):
   """
 
   def __init__(self, checkpoint, directory,
-               max_to_keep, keep_checkpoint_every_n_hours=None):
+               max_to_keep, keep_checkpoint_every_n_hours=None, checkpoint_name=None):
```

Comment:
Why None? Seems slightly clearer to just default the argument to "ckpt".

And could you add a short blurb to the Args: section of the docstring?

Reply:
Changed the default value to "ckpt" and added a desctiotion for the parameter.
Working on the API goldens...